test/index.ts

```diff
@@ -0,0 +1,74 @@
+
+import { should } from 'chai';
+import { Crypto } from '../src/';
+import Keyring from "@polkadot/keyring";
+
+should();
+
+describe('crypto', () => {
+    it('Test crypto structure', async () => {
+      // Testing result format.
+      const nodes = 5;
+      const c = new Crypto(nodes);
+
+      const result = await c.createKeys();
+      Object.keys(result).length.should.eq(c.keyTypes.length);
```

Comment:
I think we should export KeyTypes, extract it from the class. It doesn't make sense to require to instantiate the class to know which key types it can handle. We should be able to do `import { KeyTypes } from '../src/index.ts'`